src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,106 @@
+<script>
+export default {
+  props: {
+    location: {
+      type: Object,
+      required: true,
+      validator: val => val.type && val.id,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    append: {
```

Comment:
Doesn't look like this is used, what's the reason for this prop?

Reply:
It's used in #48